lib/deterministicmnlist/SimplifiedMNList.js

```diff
@@ -47,8 +49,7 @@ function SimplifiedMNList(simplifiedMNListDiff) {
  */
 SimplifiedMNList.prototype.applyDiff = function applyDiff(simplifiedMNListDiff) {
   // This will copy instance of SimplifiedMNListDiff or create a new instance if serialized data is passed
-  var diff = new SimplifiedMNListDiff(simplifiedMNListDiff);
-
+  var diff = new SimplifiedMNListDiff(simplifiedMNListDiff, this.network);
```

Comment:
is `this.network` undefined at the beginning?

Shouldn't we check that the previous diff network and a new one are equal?

Reply:
> is `this.network` undefined at the beginning?
yep, the first one can be undefined. But if it is undefined it will be set in the next line
https://github.com/dashevo/dashcore-lib/blob/quorum-verification/lib/deterministicmnlist/SimplifiedMNList.js#L53
> Shouldn't we check that the previous diff network and a new one are equal?
we already do it here (your latest commit):
https://github.com/dashevo/dashcore-lib/blob/master/lib/deterministicmnlist/SimplifiedMNListDiff.js#L185
Anything you'd like to change?

Reply:
I see. Honestly, I wouldn't say that network handling is straightforward here, but it's outside of the task's scope.